src/app/apollo-cache.helper.ts

```diff
@@ -22,17 +22,21 @@ export function apolloCache(): InMemoryCache {
                 merge: false,
             },
             Dataset: {
-                // Use owner, as ID might be the same between 2 accounts who synchronized
-                keyFields: ["owner", "id"],
                 fields: {
-                    metadata: {
-                        merge: true,
-                    },
```

Comment:
Why??? This would completely kill caching of datasets.

Reply:
I returned it back, I thought the warning was because of this - it turned out not